students/kaczmarek_katarzyna/lesson_02_flow_control/the_second_maximum.py

```diff
@@ -0,0 +1,12 @@
+max1 = int(input())
```

Comment:
użytkownik nie podaje tych liczb. Jedynie na bieżąco podaje liczby i to program ma wśród nich znaleźć właściwą.
Jeżeli użytkownik poda za duże liczby na początku to nie znajdziesz prawidłowej liczby.